syntax = "proto3";

package telemetry;

message TimeFrameInfo {
  optional int64 id = 1;
  optional string window_title = 10;
  optional string process_executable_path = 11;
  optional int64 utc_timestamp = 20;
}